source/connection-troubleshooting.txt

```diff
@@ -183,22 +174,27 @@ issues when attempting to connect to MongoDB using ``SCRAM-SHA-256``.
    see :ref:`Connection URI <node-connection-uri>` in the Connection Guide.
 
 If your connection string contains a username and password, ensure that they
-are in the correct format.
+are in the correct format. If the username or password includes any of the
+following characters, they must be
+`percent encoded <https://tools.ietf.org/html/rfc3986#section-2.1>`__:
 
-.. note::
+.. code-block:: none
 
-   If the username or password includes any of the following characters, they
-   must be `percent encoded <https://tools.ietf.org/html/rfc3986#section-2.1>`__:
+    : / ? # [ ] @
 
-   .. code-block:: none
+The following example shows how to percent encode "#MyP@assword?":
 
-      : / ? # [ ] @
+.. code-block:: javascript
+
+    const password = '#MyP@assword?';
+    const encodedPassword = encodeURIComponent(password);
+    console.log(encodedPassword);
```

Comment:
Issue:
The indentation looks off by one space.

Suggestion:

Fix the indentation. Applies to all instances.

The example also seems more verbose than necessary.

```javascript
  const encodedPassword = encodeURIComponent('#MyP@assword?');
  // The value of encodedPassword is "%23MyP%40assword%3F"
```

If you want to keep the split between code and output, this could be the code:

```javascript
  console.log(encodeURIComponent('#MyP@assword?'));
```